test/test_sbol2_sbol3_direct.py

```diff
@@ -11,15 +11,15 @@
 
 TEST_FILES = Path(__file__).parent / 'test_files'
 
-
```

Comment:
Please avoid making un-needed changes in spacing: it increases the likelihood of conflicts.
There are several locations, here and below.